.github/workflows/sbom.yml

```diff
@@ -0,0 +1,52 @@
+name: SBOM generation
+
+on:
+  push:
+  pull_request:
```

Comment:
```suggestion
on:
  schedule:
    # Runs at 02:00 UTC every Monday
    - cron: '2 0 * * 1'
```

Reply:
Not sure this is the right schedule, but why don't we just put *some* sort of arbitrary schedule in here and get this merged so we can start seeing what data we get out of it.  (And also so I can get it out of the open pull request list before hacktoberfest starts!)